tests/virtual_machines_test.go

```diff
@@ -403,6 +403,12 @@ var _ = Describe("Virtual Machines", func() {
 		})
 		//2633
 		Context("When we re-apply a failed VM yaml", func() {
+			totalTimeout := time.Duration(0)
+			BeforeAll(func() {
```

Comment:
You should use `BeforeEach` here I am almost sure `BeforeAll` invoked before test suit starts

Reply:
If I do BeforeEach thenit will get run getVmFailCleanupWaitTime before each tests - which is unnecessary since it's not changing at all.
IIUC, BeforeAll will simply do it once in the beginning of the context, but only inside `Context("When we re-apply a failed VM yaml"` context.

Reply:
You are right, [`BeforeAll` ](https://github.com/k8snetworkplumbingwg/kubemacpool/blob/29c33895384710a7d2528c70b4059ee2663e50fc/tests/tests.go#L358) its not standard BTW

Reply:
You are right, BTW [BeforeAll](https://github.com/k8snetworkplumbingwg/kubemacpool/blob/29c33895384710a7d2528c70b4059ee2663e50fc/tests/tests.go#L358) is not standard

Reply:
well the shoe fits I think :)

Reply:
didn't even notice :) well somebody ought to add it to their API asap! :)